packages/web-react/src/types/button.ts

```diff
@@ -3,17 +3,20 @@ import { AriaLabelingProps } from './shared/dom';
 import { ChildrenProps, ClickEvents, StyleProps } from './shared';
 
 export type ButtonColor = 'primary' | 'secondary' | 'tertiary' | 'inverted' | 'danger';
+export type ButtonSize = 'medium' | 'large';
 type ButtonType = 'button' | 'submit' | 'reset';
 
 interface ButtonProps extends ChildrenProps, ClickEvents {
   /** Whether the button is disabled. */
   isDisabled?: boolean;
   /** The color of the button. */
-  color: ButtonColor;
+  color?: ButtonColor;
```

Comment:
Both `ButtonColor` and `ButtonSize` have default values therefore they should not be a maybe type.

Reply:
We talked about it in Slack. Solved. ✅